.github/workflows/build.yml

```diff
@@ -0,0 +1,158 @@
+name: Test es-module-lexer
+
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    branches:
+      - 'feat/**'
+      - 'fix/**'
+      - 'main'
+  pull_request:
```

Comment:
Github actions runs not only on this repository, but also should run on the forkd one which PR was taken from. Developers could make one branch for PR, which has name started with `feat/` for new feature or `fix/` for some fixup, just like what I do this time. 

So while rule for pushing/PR-to `main` branch was kept, I still leave two special rules :

- `feat/**`
- `fix/**`

@guybedford